src/config/ThemeProvider.tsx

```diff
@@ -12,7 +12,6 @@ export interface ThemeProps<T> {
   replaceTheme: (updates: RecursivePartial<FullTheme>) => void;
 }
 
-// @ts-ignore
 export const ThemeContext: React.Context<ThemeProps<{}>> = React.createContext({
```

Comment:
@pranshuchittora sir here i tried to remove ThemeProps interface which result in test faliure as this components is utilized in withTheme components can you please provide me clue to resolve error .